OutOfSchool/OutOfSchool.DataAccess/Repository/IAverageRatingRepository.cs

```diff
@@ -0,0 +1,11 @@
+﻿using OutOfSchool.Services.Enums;
+using OutOfSchool.Services.Models;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace OutOfSchool.Services.Repository;
+
+public interface IAverageRatingRepository : IEntityRepository<long, AverageRating>
```

Comment:
this repository and interface have no methods/implementation. This is covered automatcally by using `IEntityRepository<long, AverageRating>` imported type instead of custom type (no need to add any method call or class). Please remove `AverageRatingRepository` and `IAverageRatingRepository` unless you plan to add methods here.

Reply:
The interface and the repository were removed.